internal/config/root.go

```diff
@@ -74,9 +74,9 @@ func (c *Root) init() error {
 	return nil
 }
 
-func (c *Root) SandboxDashboardURL(id string) *url.URL {
+func (c *Root) SandboxDashboardURL(name string) *url.URL {
 	u := *c.DashboardURL
-	u.Path = path.Join(u.Path, "sandbox", "id", id)
+	u.Path = path.Join(u.Path, "sandbox", "name", name)
```

Comment:
Have you verified that this generates a valid URL?

Reply:
Yes, 

![Screenshot 2024-08-22 at 4 57 39 PM](https://github.com/user-attachments/assets/474265a8-c796-4ea8-8e24-9edd03691b11)

```
ID:               t16m5ym2qtf54
Name:             new-series-driver
Description:      Change driver series
Cluster:          demo
Created:          Wed, 21 Aug 2024 14:31:35 -05 (26 hours ago)
Updated:          Wed, 21 Aug 2024 14:31:35 -05 (26 hours ago)
TTL:              - (forever)
Dashboard page:   https://app.signadot.com/sandbox/name/new-series-driver
Status:           Ready (AllDesiredWorkloadsReady: All desired workloads are available.)

```